packages/gwt-editors/src/GwtEditorWrapper.tsx

```diff
@@ -92,4 +97,10 @@ export class GwtEditorWrapper extends AppFormer.Editor {
       headerTable.remove();
     }
   }
+
+  private injectStyleToFixResponsivenessIssue_DROOLS_3995() {
+    const style = document.createElement("style");
+    style.textContent = ".list-view-pf-body { display: flex !important; }";
```

Comment:
@tiagobento Since it's applied globally for `.list-view-pf-body`, it can potentially change the behaviour in every use of `.list-view-pf-body`.

A safer possible alternative would be this:
```suggestion
    style.textContent = "[data-i18n-prefix=\"DataTypeListItemView.\"] .list-view-pf-body { display: flex !important; }";
```

Reply:
That's great, @karreiro! Thanks for the suggestion. To be even better, I think we could replace the string's double quotes by single quotes and avoid using the escaping slash :)

Reply:
@tiagobento I've updated the _Suggested change_ following your suggestion ;-)